Documentation/QRCodeSign.md

```diff
@@ -0,0 +1,28 @@
+**Instructions**
+
+ All flowers are labeled, allowing visitors to take note of top performing cultivars under regional conditions.
+ 
+ <br>
+ 
+**Search for your favorite flower**
+
+ There are three ways to get to the flower you are looking for:
+ 1. Scan the QR code located by each of the beds
+ 2. Enter the common name (e.g. Petunia) in the search box to view a list of all that type of flower along with their corresponding location in the Garden
+ 3. Select a garden bed from the drop-down menu and view the list of all flowers in that particular garden bed.
+
+_Each growing season, hundreds of annual flower cultivars from plant breeding companies around the world are trialed at the WCROC. Cultivars are grown from seed or are vegetatively propagated in our greenhouse. Most of the plants you'll find in the Horticulture Display Garden are under evaluation. All flowers are labeled, allowing visitors to take note of top performing cultivars under regional conditions._
+___
+<br>
+<br>
+<br>
+<br>
+<br>
+
+**How to scan QR codes:**
+1. Download a QR scanner app on your smartphone
```

Comment:
Should we give a recommendation of what Apps to use? I use [Barcode Scanner](https://play.google.com/store/apps/details?id=com.google.zxing.client.android&hl=en) on Android and really like it, but I don't know if anyone on our team found one that they really like for iOS. So maybe just ignore me.